### ISSUE TITLE
Some Means of Removing Stuck Furniture/Craftables

### ISSUE THREAD

**Issue by StardewSweetheart** [StardewModders/mod-ideas#1065]
I placed this torch after grabbing it from CJB Item Spawner but it decided to stack itself endlessly (hence the super dark shadow under it) after disappearing from the map a few times.

<img width="329" alt="Screen Shot 2023-02-19 at 12 23 58 PM" src="https://user-images.githubusercontent.com/106191569/219973320-79915dcf-a141-4451-acd1-f7ded4296de6.png">  <img width="200" alt="Screen Shot 2023-02-19 at 12 23 41 PM" src="https://user-images.githubusercontent.com/106191569/219973363-341bb7d6-1e6b-4aac-8082-62c526e41f5d.png">

The problem is that I can't get rid of it by clicking on it or by hitting it with an axe or pickaxe. Reset Terrain doesn't work on furniture/craftables. I can't tell if this particular torch is bugged or if it's something else. Any means of removing it (outside of doing a new save) would be much appreciated.

**Comment by aedenthorn** [comment]
maybe try the mod called **Modify This**.

**Comment by StardewSweetheart** [comment]
Nope, unfortunately Modify This doesn't recognize as a valid item either. :(

**Comment by StardewSweetheart** [comment]
Ok, I thought maybe it's because Modify This doesn't yield a menu to work on anything that has an Alternative Texture, which Plain Torch did until I took out its folder, but that didn't fix the issue. :(

**Comment by aedenthorn** [comment]
try the new version of Mod This, it might be furniture

**Comment by StardewSweetheart** [comment]
I think it is, but unfortunately the newest version of Mod This still won't let me interact with it or other pieces of furniture yet. 

I'm also having a new issue wherein picking up a furniture item I placed down yields a stack of that object, not a single one. This happens without Mod This installed so I am going to try and troubleshoot that separately. I'll stop by your Discord server so we can chat there if that's ok. :)

**Comment by tiakall** [comment]
This is more a troubleshooting request than a mod idea, but Let's Move It or a debug command might fit the bill.
